db/seeds.d/07-provisioning_templates.rb

```diff
@@ -95,6 +95,8 @@
     contents = File.read(File.join("#{Rails.root}/app/views/unattended", input.delete(:source)))
 
     if (t = ProvisioningTemplate.unscoped.find_by_name(input[:name])) && !audit_modified?(ProvisioningTemplate, input[:name])
+      next if t.global_default?
```

Comment:
I think, we should remove this when we actually implement the new workflow (i.e. lock the shipped templates).